tools/pylint-extensions/azure-pylint-guidelines-checker/pylint_guidelines_checker.py

```diff
@@ -1346,11 +1383,22 @@ def check_parameters(self, node):
             if param not in docparams:
                 missing_params.append(param)
 
+        # check that all keyword-only args are documented
+        missing_kwonly_args = []
```

Comment:
check that all code-defined keyword-only args are in docstring. Any difference --- raise error